build/alpine/Dockerfile.deps

```diff
@@ -27,4 +27,5 @@ RUN apk add --no-cache \
             llvm-libunwind \
             ca-certificates \
             dumb-init \
+            xz \
```

Comment:
I looked for liblzma.so.5 and found it in `xz-utils`:
https://pkgs.alpinelinux.org/contents?file=liblzma.so.5&path=&name=&branch=v3.12
TBH, I don't really know the difference between liblzma.so and liblzma.so.5, but my locally compiled binaries depend on `liblzma.so.5`.

Reply:
`xz-libs` is what you meant to write, yes, I added that (though it seems to already get included by something else and is present already in the alpine-deps image).

Reply:
> xz-libs is what you meant to write
yes, not enough coffee!

Did you see my other comment? Maybe we don't need this PR, the pipeline is happy.

Reply:
I thought explicit is better than not explicit, but ok :)